src/cli.js

```diff
@@ -145,14 +163,16 @@ program
       );
       progressBar.start(documents.count);
 
+      const slugPrefixLC = folderToSlugLC(folder);
```

Comment:
such as `web/css/_colon_host`, the slug should be `web/css/:host` (in lowercase)